docs/requirements.md

```diff
@@ -1,51 +1,140 @@
 
-### Confirm that the Cassandra-compatible database supports lightweight transactions (LWTs)
+</div>
 
-You must use a Cassandra-compatible database that supports LWTs.
+<div id="Cosmos_DB_for_NoSQL" class="tabcontent" markdown="1">
 
-ScalarDB does not work on some Cassandra-compatible databases that do not support LWTs, such as [Amazon Keyspaces](https://aws.amazon.com/keyspaces/). This is because the Consensus Commit transaction manager relies on the linearizable operations of underlying databases to make transactions serializable.
+#### Transactions
+{:.no_toc}
+- Use a single primary region for all operations with `Strong` or `Bounded Staleness` consistency.
 
-## CosmosDB database requirements
+#### Analytics
+{:.no_toc}
+- N/A (Thee are no Cosmos DB-specific requirements since Cosmos DB always returns committed records.)
 
-In your Azure CosmosDB account, you must set the **default consistency level** to **Strong**.
+</div>
 
-Consensus Commit, the ScalarDB transaction protocol, requires linearizable reads. By setting the **default consistency level** to **Strong**, CosmosDB can guarantee linearizability.
+<div id="Cassandra" class="tabcontent" markdown="1">
 
-For instructions on how to configure this setting, see the official documentation at [Configure the default consistency level](https://learn.microsoft.com/en-us/azure/cosmos-db/nosql/how-to-manage-consistency#configure-the-default-consistency-level).
+#### Transactions
+{:.no_toc}
+- Use a single primary cluster for all operations. (No reads and writes in non-primary clusters.)
+- Use `batch` or `group` for `commitlog_sync`.
+- (In case you use Cassandra-compatible databases) Use Cassandra-compatible databases that properly support Lightweight Transactions (LWT).
 
-## JDBC database recommendations
+#### Analytics
+{:.no_toc}
+- N/A (Thee are no Cassandra-specific requirements since Cassandra always returns committed records.)
 
-In ScalarDB on JDBC databases, you can't choose a consistency level (`LINEARIZABLE`, `SEQUENTIAL` or `EVENTUAL`) in your code by using the `Operation.withConsistency()` method. In addition, the consistency level depends on the setup of your JDBC database.
+</div>
+</div>
 
-For example, if you have asynchronous read replicas in your setup and perform read operations against them, the consistency will be eventual because you can read stale data from the read replicas. On the other hand, if you perform all operations against a single master instance, the consistency will be linearizable.
+## Recommendations
 
-With this in mind, you must perform all operations or transactions against a single master instance so that you can achieve linearizability and avoid worrying about consistency issues in your application. In other words, ScalarDB does not support read replicas. 
+ScalarDB recommends each underlying database to be configured properly for high performance and high availability. The following items are some knobs and configurations to update.
```

Comment:
In this case, ScalarDB can't really `recommend` (it can `require`, however), so I've re-worded this a bit to use passive voice since using active voice would be a little awkward.

```suggestion
Properly configuring each underlying database of ScalarDB for high performance and high availability is recommended. The following recommendations include some knobs and configurations to update.
```